cg/apps/cgstats/crud/create.py

```diff
@@ -154,7 +154,7 @@ def create_dragen_unaligned(
 
 def _calculate_perfect_indexreads_pct(demux_sample: DragenDemuxSample) -> float:
     """calculates the percentage of perfect index reads"""
-    return round(demux_sample.perfect_reads / demux_sample.reads * 100, 2)
+    return round(demux_sample.perfect_reads / demux_sample.reads * 100, 2) if demux_sample.reads else 0
 
 
 def _calculate_q30_bases_pct(demux_sample: DragenDemuxSample) -> float:
```

Comment:
I agree with @moahaegglund that we should make the same changes to this function @barrystokman

Reply:
This should be for line 160 `_calculate_q30_bases_pct`

Reply:
I saw that too, it's incoming :)